hooks/VDM.js

```diff
@@ -305,303 +305,229 @@ module.exports = function (input, session) {
-        path: `environment.observations.${meteoLocation}.air.dewPoint`,
-        value: utils.transform((data.dewpoint / 10), 'c', 'k'),
-      })
-    }
+    values.push({
+      path: `environment.air.dewPoint`,
+      value: utils.transform(data.dewpoint, 'c', 'k'),
+    })
   }
 
   if (data.airpress) {
     contextPrefix = 'meteo.'
-    if (data.airpress < 403) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.pressure`,
-        value: (data.airpress + 799) * 100,
-      })
-    }
+    values.push({
+      path: `environment.air.pressure`,
+      value: data.airpress * 100,
+    })
+
   }
 
   if (data.airpressten) {
     contextPrefix = 'meteo.'
-    if (data.airpressten < 3) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.pressureTendency`,
-        value: statusTable[data.airpressten],
-      })
-    }
+    values.push({
+      path: `environment.air.pressureTendency`,
+      value: statusTable[data.airpressten],
+    })
   }
 
   if (data.horvisib) {
     contextPrefix = 'meteo.'
-    if (data.horvisib < 127) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.horizontalVisibility`,
-        value: utils.transform((data.horvisib / 10), 'nm', 'm'),
-      })
-    }
+    values.push({
+      path: `environment.air.horizontalVisibility`,
+      value: utils.transform(data.horvisib, 'nm', 'm'),
+    })
   }
 
   if (data.waterlevel) {
     contextPrefix = 'meteo.'
-    if (data.waterlevel < 4001) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.level`,
-        value: (data.waterlevel / 100) - 10,
-      })
-    }
+    values.push({
+      path: `environment.water.level`,
+      value: data.waterlevel,
+    })
   }
 
   if (data.waterlevelten) {
     contextPrefix = 'meteo.'
-    if (data.waterlevelten < 3) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.levelTrend`,
-        value: statusTable[data.waterlevelten],
-      })
-    }
+    values.push({
+      path: `environment.water.levelTrend`,
+      value: statusTable[data.waterlevelten],
+    })
   }
 
   if (data.surfcurrspd) {
     contextPrefix = 'meteo.'
-    if (data.surfcurrspd < 252) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.surfaceCurrentSpeed`,
-        value: utils.transform((data.surfcurrspd / 10), 'knots', 'ms'),
-      })
-    }
+    values.push({
+      path: `environment.water.surfaceCurrentSpeed`,
+      value: utils.transform(data.surfcurrspd, 'knots', 'ms'),
+    })
   }
 
   if (data.surfcurrdir) {
     contextPrefix = 'meteo.'
-    if (data.surfcurrdir < 360) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.water.surfaceCurrentDirection`,
-        value: utils.transform(data.surfcurrdir, 'deg', 'rad'),
-      })
-    }
+    values.push({
+      path: `environment.water.surfaceCurrentDirection`,
```

Comment:
To align with `environment` group in the spec use `environment.observations.current` instead.